invenio_rdm_records/config.py

```diff
@@ -338,4 +304,118 @@ def always_valid(identifier):
         "validator": always_valid
     }
 }
-RDM_RECORDS_DOI_DATACITE_FORMAT = "{prefix}/{id}"
+
+
+#
+# Record permission policy
+#
+RDM_PERMISSION_POLICY = None
```

Comment:
Intended to be set from ``invenio.cfg`` if you want to override it.

Reply:
I purposedly name the variables ``RDM_*`` instead of ``RDM_RECORDS_*``. These variables are intended to be overwritten in ``invenio.cfg`` by end-users, and long configuration variable names are not very user friendly.

We should align on existing ones as well.

Reply:
For maintainers, this naming convention works for navigating the code, but I think it fails for users who just install the system and need to configure it. I think for these users, feature-wise grouping in the naming works well. E.g. the feature of search is split over both App-RDM, RDM-Records and more, but that doesn't matter for the user - for them it's just search. 

One possible solution is naming everything in ``App-RDM`` and ``RDM-Records`` with the ``RDM_`` prefix.